cvat-ui/src/components/annotation-page/standard3D-workspace/controls-side-bar/photo-context.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import Icon from '@ant-design/icons';
+import Tooltip from 'antd/lib/tooltip';
+
+import { CameraIcon } from 'icons';
+import { ActiveControl } from 'reducers/interfaces';
+import { Canvas3d as Canvas } from 'cvat-canvas3d-wrapper';
+
+interface Props {
+    canvasInstance: Canvas;
+    activeControl: ActiveControl;
+    hideShowContextImage: any;
```

Comment:
Let's use more specific type instead of any. I believe it is possible here.

Reply:
hideShowContextImage is a method, hence used any.

Reply:
It's type could be ``(hide: boolean) => void``, not ``any``.

Reply:
Resolved